From 46b3079095c5ceb0dc742785853bbaf288f325c6 Mon Sep 17 00:00:00 2001
From: Joe Slater <jslater@windriver.com>
Date: Wed, 25 Jan 2017 13:52:59 -0800
Subject: [PATCH] gnutls: account for ARM_EABI

Certain syscall's are not availabe for arm-eabi, so we eliminate
reference to them.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>
---
 tests/seccomp.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/seccomp.c b/tests/seccomp.c
index 881f0bb..5f9204a 100644
--- a/tests/seccomp.c
+++ b/tests/seccomp.c
@@ -55,7 +55,9 @@ int disable_system_calls(void)
 
 	ADD_SYSCALL(nanosleep, 0);
 	ADD_SYSCALL(clock_nanosleep, 0);
+#if ! defined(__ARM_EABI__)
 	ADD_SYSCALL(time, 0);
+#endif
 	ADD_SYSCALL(getpid, 0);
 	ADD_SYSCALL(gettimeofday, 0);
 #if defined(HAVE_CLOCK_GETTIME)
